src/js/orders.spec.ts

```diff
@@ -24,7 +24,7 @@ export class Order {
     this.nonce = new BN(nonce);
   }
 
-  encode(): string {
+  encode(): Buffer {
```

Comment:
Weird that this changed, but nothing else had to be adjusted... was this one of the warnings?

Reply:
I imported ethereumjs-abi, and hence it needed to be corrected